src/integrationTest/resources/tests/dailylist/DailyListServiceTest/insert1_ok/expectedResponse.json

```diff
@@ -20,5 +20,7 @@
   "contentObjectId": null,
   "clipId": null,
   "externalLocation": null,
-  "externalLocationTypeEntity": null
+  "externalLocationTypeEntity": null,
+  "subcontentObjectId": null,
+  "subcontentPosition": null
```

Comment:
Should this be in snake_case to follow HMCTs standards if they are new fields?

Reply:
So this isn't actually an API response (despite the name of the file). Some tests take the approach of using a JSON file to represent the expected state of an object, and then serialising the actual object and comparing it to the expected JSON with `JSONAssert`. I'm not a fan of the approach personally.

So here, all of the attribute names actually correspond to the attribute names of a `DailyListEntity`.